src/classes/CanvasRenderingContext2D.js

```diff
@@ -637,13 +625,14 @@ export default class CanvasRenderingContext2D {
   set fillStyle(value) {
     let valid = false;
     if (typeof value === 'string') {
-      const result = parseCSSColor(value);
-
-      if (result) {
+      try {
+        const result = new MooColor(value);
         valid = true;
-        value = this._fillStyleStack[this._stackIndex] = result;
+        value = this._fillStyleStack[this._stackIndex] = result.toHex();
```

Comment:
Suggesting to store the color as the correct string value here.

`serializeColor(result)`

Reply:
Reason: The browser parses the color with every `set` operation, and we would be doing exactly the same thing.